dukan/src/main/kotlin/net/thechance/dukan/entity/Dukan.kt

```diff
@@ -22,7 +22,7 @@ data class Dukan(
     )
     val categories: Set<DukanCategory>,
     @Column(name = "image_url", nullable = true)
-    val imageUrl: String? = null,
+    var imageUrl: String? = null,
```

Comment:
Please keep the table columns immutable

```suggestion
    val imageUrl: String? = null,
```

Reply:
done